parser/tests/Create.test

```diff
@@ -139,6 +139,74 @@ CREATE TABLE test (attr INT) CHECK(attr>0)
                              ^
 ==
 
+CREATE TABLE test (attr INT) PARTITION BY HASH(attr) PARTITIONS 4
+--
+CreateTableStatement[relation_name=test]
++-attribute_list=
+| +-AttributeDefinition[name=attr,type=Int]
++-partition_clause=
+  +-PartitionClause[partition_type=0]
+    +-Number of Partitions=NumericLiteral[numeric_string=4,float_like=false]
+    +-attribute_name_list=
+      +-String[value=attr]
+==
+
```

Comment:
OK. I've resumed the build.

Then the PR title should be `parser support for Hash Partition`.